### PR TITLE
Optimize test coverage of all lines of Custom Components

### PR DESCRIPTION
**Pull request by oreoluwade**
#### What does this PR do?
- It ensures that all lines of the custom components are covered while testing.
- It optimizes the formValidation helper test-suite.

#### Description of Task to be completed?
- Put the unused props of the custom components to use.
- Get rid of redundant props.
- Update utils test suite.
- Add default props for `Text` component.

#### How should this be manually tested?
- Run `yarn test` and notice that all lines of `Box`, `Text` and `Button` components are covered.

#### Any background context you want to provide?
- Failure to use certain props at all in the codebase ensures they aren't tested and affects the overall test coverage. Ensuring that all lines are covered in testing ensures that every unnecessary chunk of code is quickly scrapped and that unforeseen cases are adequately handled.

#### What are the relevant pivotal tracker stories?
[#154021840](https://www.pivotaltracker.com/story/show/154021840)

### CODE REVIEW

File: src/components/CustomComponents/Text.js [hunk 1]
```diff
@@ -55,6 +54,12 @@ const Text = (props) => {
   if (props.uppercase) {
     inlineStyles.textTransform = 'uppercase';
   }
+  if (props.defaultFont) {
+    inlineStyles.fontFamily = 'Garamond';
```
Comment (fisayomi) [inline src/components/CustomComponents/Text.js]: Is it possible to use the globally injected font family for all Custom Components?

Comment (oreoluwade) [reply src/components/CustomComponents/Text.js]: @fisayomi, the way I see it, we mostly initialize the styles of our custom components to an empty object, and then we start defining rules, one after the other to populate the style object. I will try to investigate this in another story to see how we can ensure that any component accepts certain `body-css-rules`.

File: src/apollo-client/apollo-client.js [hunk 2]
```diff
@@ -15,7 +16,13 @@ const middlewareLink = new ApolloLink((operation, forward) => {
       authorization: localStorage.getItem('token') || null,
     },
   });
-  return forward(operation);
+  // log operations as they are fired
+  console.log(`OPERATION NAME: ${operation.operationName}`);
```
Comment (omorilewa) [inline src/apollo-client/apollo-client.js]: @oreoluwade Do you think this is a good idea in production? For a nosy user, we may be sharing too much information.

Comment (oreoluwade) [reply src/apollo-client/apollo-client.js]: @omorilewa Good point. I will see if I can ensure this happens only in `dev`.

**Pull request merged**